### PR TITLE
[TR#47] add slim layouts and change to use slim + add bootstrap

### PR DESCRIPTION
**Pull request by Malachirwin**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,12 +5,11 @@ gem 'puma',         '3.9.1'
 gem 'sass-rails',   '5.0.6'
 gem 'uglifier',     '3.2.0'
 gem 'coffee-rails', '4.2.2'
-gem 'jquery-rails', '4.3.1'
 gem 'turbolinks',   '5.0.1'
 gem 'jbuilder',     '2.7.0'
 gem 'rspec-rails'
 gem 'slim-rails'
-gem 'bootstrap-sass', '3.3.7'
+gem 'bootstrap', '~> 4.0.0.beta.3'
```
Comment (joshfaggion) [inline Gemfile:12]: Do we need to specify such a specific version?

File: app/assets/stylesheets/_bootstrap-custom.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/*!
+ * Bootstrap v4.0.0-beta.3 (https://getbootstrap.com)
+ * Copyright 2011-2017 The Bootstrap Authors
+ * Copyright 2011-2017 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+@import "bootstrap";
```
Comment (joshfaggion) [inline app/assets/stylesheets/_bootstrap-custom.scss:7]: Good job!

**Review by joshfaggion — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/_bootstrap-custom.scss (hunk 2 above)
Comment (joshfaggion) [reply app/assets/stylesheets/_bootstrap-custom.scss:7]: Once you answer the question, we are good to go.

File: Gemfile (hunk 1 above)
Comment (Malachirwin) [reply Gemfile:12]: That is what ninja master had

**Comment by joshfaggion** [comment]
I'm going to add a trello card to see if the version needs to be so specific, but this is going to be approved.

**Review by joshfaggion — APPROVED** [review]

**Pull request merged**